pom.xml

```diff
@@ -2036,6 +2037,12 @@
                 <version>${mock-server.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.jeasy</groupId>
+                <artifactId>easy-random-core</artifactId>
+                <version>${jeasy.version}</version>
```

Comment:
Again, Easy Random has 3 years with no releases. I will be happy in the test scope until it works.